pkgs/development/interpreters/cg3/default.nix

```diff
@@ -0,0 +1,41 @@
+{ lib
+, stdenv
+, fetchurl
+, cmake
+, icu
+, boost
+}:
+
+stdenv.mkDerivation rec {
+  pname = "cg3";
+  version = "1.3.7";
+
+  src = fetchurl {
+    url = "https://github.com/GrammarSoft/${pname}/releases/download/v${version}/${pname}-${version}.tar.bz2";
+    sha256 = "d8d996c60063baae2be1088d6122678179ab3c47653e76edd6cda07a423f4660";
+  };
+
+  nativeBuildInputs = [
+    cmake
+  ];
+
+  buildInputs = [
+    icu
+    boost
+  ];
+
+  # cmakeFlags = [
+  #   "-DCMAKE_INSTALL_PREFIX=${prefix}/"
+  # ];
+
+  # TODO: tcmalloc unless darwin?
+  # option(ENABLE_PYTHON_BINDINGS "Set to ON to build the Python wrapper" OFF)
+
+  meta = with lib; {
+    homepage = "https://github.com/GrammarSoft/cg3";
+    description = "Constraint Grammar interpreter, compiler and applicator vislcg3";
+    maintainers = [ lib.maintainers ];
```

Comment:
A copy/paste mistake? This should list actual maintainers.

Reply:
Oh I wasn't sure about that one. Should I add myself to maintainers/maintainers-list.nix?

Reply:
Yes, please. The PR should be then 2 commits - one to add yourself, another with all the package changes.

Reply:
:heavy_check_mark: